spec/steps/bookings_steps.rb

```diff
@@ -16,6 +16,11 @@
   end
 end
 
+step 'I cannot see the until date field when is selected the no repeat option' do
+  expect(page).to have_select('Repeat event', selected: 'No repeat')
+  expect(page).to have_css('.js-dynamic-form-element-to-hide', visible: false)
```

Comment:
The selector you use here is not safe. What if we add another dynamic field to the form in the future?

Reply:
You're right. I think that I could add a new data-attribute to explicitly declare the class that we'll use to hide the field.